packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/selectBox.tests.js

```diff
@@ -6481,6 +6481,45 @@ QUnit.module('valueChanged handler should receive correct event', {
     });
 });
 
+QUnit.module('onKeyDown', () => {
+    QUnit.test('should fire event only once (T1238121)', function(assert) {
+        const onKeyDownHandler = sinon.spy();
+        const products = [{
+            ID: 1,
+            Name: 'HD Video Player',
+            ImageSrc: 'https://js.devexpress.com/jQuery/Demos/WidgetsGallery/JSDemos/images/products/1-small.png',
```

Comment:
the data can be simplified i suppose, no need to use imagerSrc etc, yep fieldTemplate is required, but it doesn't have to use these fields

Reply:
[c6abdf1](https://github.com/DevExpress/DevExtreme/pull/27660/commits/c6abdf1778ac0e1304874cd6ad33e45c7c57a307)